charts/loki-stack/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: "v1"
 name: loki-stack
-version: 2.3.1
-appVersion: v2.1.0
+version: 2.4.0
```

Comment:
I'd bump the major version since at least the promtail chart had a breaking change.